datascience/tables.py

```diff
@@ -1328,98 +1328,110 @@ def with_rows(self, rows):
         return self
 
     def with_column(self, label, values):
-        """Return a table with an additional or replaced column.
+        """Return a new table with an additional or replaced column.
 
         Args:
             ``label`` (str): The column label. If an existing label is used,
-                that column will be replaced in the returned table.
+                the existing column will be replaced in the new table.
 
             ``values`` (single value or sequence): If a single value, every
-                value in the new column is ``values``.
+                value in the new column is ``values``. If sequence of values, 
+                new column takes on values in ``values``. 
 
                 If a sequence, the new column contains the values in
-                ``values``. ``values`` must be the same length as the table.
+                ``values``, must be the same length as the table.
 
         Raises:
             ``ValueError``: If
                 - ``label`` is not a valid column name
-                - ``values`` is a list/array and does not have the same length
+                - if ``label`` is not of type (str) 
+                - ``values`` is a list/array that does not have the same length
                   as the number of rows in the table.
 
-        >>> tiles = Table().with_columns([
-        ...     'letter', ['c', 'd'],
-        ...     'count',  [2, 4],
-        ... ])
-        >>> tiles.with_column('points', [3, 2])
-        letter | count | points
-        c      | 2     | 3
-        d      | 4     | 2
-        >>> tiles.with_column('count', 1)
+        Returns:
+            copy of original table with new or replaced column
+
+        >>> alphabet
```

Comment:
The python code in the docstring actually gets run as doctests (https://docs.python.org/3/library/doctest.html), so this line makes the build fail (https://travis-ci.org/data-8/datascience/builds/164195943) since the `alphabet` table isn't defined.

You'll want to create it using `with_columns` first.